src/Logger/DebugFileLogger.php

```diff
@@ -123,7 +123,7 @@ private function getResultsLine(
 
         $separateSections = false;
 
-        foreach ($executionResults as $index => $executionResult) {
+        foreach ($executionResults as $executionResult) {
```

Comment:
may be an idea of a mutation?

Reply:
this mutation will in most cases be killed because of "undefined variable", no?

Reply:
That, unless `$index` in unused.

Reply:
Yeah would be nice to find a way to not blindly do it which would create too many errors. So a potential idea to explore but may not be doable